### ISSUE TITLE
databricks automl preprocess drops all feature columns - how to prevent?

### ISSUE THREAD

**Issue by alisentuerk** [databricks/automl#121]
Hi, i would like to prevent databricks automl preprocess method to drop my features even if they may not contain relevant information.

`
        summary = automl.classify(train_df, target_col="label", timeout_minutes=5)

        help(summary)
        model_uri = summary.best_trial.model_path
        model = mlflow.sklearn.load_model(model_uri) #sklearn`

this is the way how i start the automl training. How can i integrate the code for it? I have not found any documentation on that.
